src/assets/scripts/client/AppController.js

```diff
@@ -188,8 +185,8 @@ export default class AppController {
         UiController.init(this.$element);
 
         this.canvasController = new CanvasController(this.$canvasesElement, this.aircraftController, this.scopeModel);
-        this.aircraftCommander = new AircraftCommander(this.aircraftController, this.aircraftController.onRequestToChangeTransponderCode);
```

Comment:
Actually, if we're treating it as a sub-part of the aircraftController, it seems fine to do this-- we can still use it via `AppController.aircraftController.aircraftCommander` (when the .getAircraftCommander() method is changed to a getter, anyway).

Resolving this one.

Reply:
@erikquinn  I removed it because it is no longer used anywhere. It doesn't need to be passed into the `InputController` anymore because it can be retrieved from the `AircraftController` with a getter (after comments above), and it is now instantiated by the `AircraftController`. Nothing else in the `AppController` appeared to need a reference to it.